packages/ui/src/components/Timeline/TimelinePoint.tsx

```diff
@@ -26,13 +26,15 @@ export interface FlowbiteTimelinePointTheme {
 export interface TimelnePointProps extends ComponentProps<"div"> {
   icon?: FC<ComponentProps<"svg">>;
   theme?: DeepPartial<FlowbiteTimelinePointTheme>;
+  render?: () => React.ReactElement;
 }
 
 export const TimelinePoint: FC<TimelnePointProps> = ({
   children,
   className,
   icon: Icon,
   theme: customTheme = {},
+  render = undefined,
```

Comment:
the default to `undefined` is redundant in this case

Reply:
🙈